content/get-started/learning-about-github/about-github-advanced-security.md

```diff
@@ -36,10 +36,35 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
 - **Security overview** - Review the security configuration and alerts for an organization and identify the repositories at greatest risk. For more information, see "[About the security overview](/code-security/security-overview/about-the-security-overview)."
 {% endif %}
 
+{% ifversion fpt %}
```

Comment:
This sentence can be used for both FPT and GHEC so suggesting the following (also used the variable for GitHub Advanced Security):
```suggestion
{% ifversion fpt or ghec%}
The table below summarizes the availability of {% data variables.product.prodname_GH_advanced_security %} features for public and private repositories.

{% elsif fpt %}
```